src/main/resources/application-it.yml

```diff
@@ -59,7 +59,13 @@ kontakt-lon-pension:
   family-id: 121
   priority: MEDIUM
   category: SALARY
-  type: PENSION
+  type: OTHER
+tjugofem-ar-pa-jobbet:
```

Comment:
Swedish? The java code defines this as "twentyfiveatwork".
Maybe stick to the same language here?

Reply:
For being consistent with the other services and easy identify corresponding OpenE-service.